packages/external-db-config/lib/service/config_reader.spec.js

```diff
@@ -12,63 +12,70 @@ describe('Config Reader Client', () => {
         const actual = await env.configReader.readConfig()
 
         expect( actual ).toEqual(ctx.config)
-    })
+    }) 
 
     test('status call will return successful message in case config is valid', async() => {
         driver.givenValidConfig()
         driver.givenValidCommonConfig()
 
         const actual = await env.configReader.configStatus()
 
-        expect( actual ).toEqual('External DB Config read successfully')
+        expect( actual.missingRequiredSecretsKeys ).toEqual([])
+        expect( actual.missingRequiredEnvs ).toEqual([])
+        expect( actual.validType ).toBeTruthy()
+        expect( actual.validVendor ).toBeTruthy()
```

Comment:
Maybe instead :

`expect(env.configReader.configStatus()).resolves.toEqual(validConfigResponse)`

And replace `validConfigResponse` with matcher that will return an object with all of these.